functions/menu.bash

```diff
@@ -175,8 +182,6 @@ show_main_menu() {
   elif [[ "$choice" == "60"* ]]; then
     choosenComponents=$(whiptail --title "Manual/Fresh Setup" --checklist "Choose which system components to install or configure:" 20 116 13 --cancel-button Back --ok-button Execute \
     "62 | Packages"               "Install needed and recommended system packages " OFF \
-    "63 | Zulu OpenJDK 32-bit"    "Install Zulu OpenJDK Java 8 32-bit" OFF \
```

Comment:
why remove ?  60 menu is supposed to offer *all* options so you can select & install them in one go.